src/Nethermind/Nethermind.Trie/VerkleTree.cs

```diff
@@ -220,15 +244,15 @@ public byte[] GetTreeKeyForStorageSlot(Address address, UInt256 storageKey)
         
     public void SetCode(Address address, byte[] code)
     {
-        byte[][] chunkifiedCode = chunkifyCode(code);
+        byte[][] chunkifiedCode = ChunkifyCode(code);
         byte[] chunkKey;
         for (int i = 0; i < chunkifiedCode.Length; i++)
         {
             chunkKey = GetTreeKeyForCodeChunk(address, (UInt256)i);
             SetValue(chunkKey, chunkifiedCode[i]);
         }
     }
-    private byte[][] chunkifyCode(byte[] code)
+    private byte[][] ChunkifyCode(byte[] code)
```

Comment:
This needs rework not to allocate array of arrays. Maybe method like:
Span<byte> TryGetNextChunk(byte[] code, ref currentPosition)

And usage:
int currentPosition = 0;
do
{
  var chunk = TryGetNextChunk(code, ref currentPosition);
  chunkKey = GetTreeKeyForCodeChunk(address, (UInt256)i);
  SetValue(chunkKey, chunk);
}
while (currentPosition < code.Length)